gusb/gusb-context.c

```diff
@@ -135,9 +119,9 @@ g_usb_context_dispose (GObject *object)
 
 static void
 g_usb_context_get_property (GObject    *object,
-                            guint       prop_id,
-                            GValue     *value,
-                            GParamSpec *pspec)
+			    guint       prop_id,
```

Comment:
are you sure about these changes? alignment vs indentation
alignment should always be with spaces IMHO

Reply:
The rest of the code uses greedy 8-space tabs. I think it's more important to be consistent.